src/app/api/utils/auth.ts

```diff
@@ -66,7 +72,9 @@ export const authOptions: AuthOptions = {
     async jwt({ token, account, profile, trigger }) {
       if (trigger === "update") {
         // Refresh the user data from FxA, in case e.g. new subscriptions got added:
-        profile = await fetchUserInfo(token.subscriber?.fxa_access_token ?? "");
+        const subscriber = await getSubscriberByEmail(token.email ?? "");
+        profile = subscriber.fxa_profile_json as FxaProfile;
```

Comment:
Is it possible `getSubscriberByEmail()` can return `null`/`undefined` and then `subscriber` is nullish below?
I'm a bit unclear on what would happen if `token.email` _is_ nullish and falls back to "".